### DISCUSSION TITLE
Can't find external file table on FlameRobin - had to go back to release 0.9.3.3

### DISCUSSION THREAD

**Discussion by FabricioAraujoRJ** [mariuz/flamerobin#175] [General]
I have an external file table (used to debug stored procedures mainly)
that do not appear on treeview in all releases post 0.9.3.3 (installer flamerobin-0.9.3.43407334-setup.exe).
This table simple doesn't appear in treeview (but I can see query it in SQL). That's a bug or I'm missing some setting? 
Server info: 
**Server Version Info**
**---------------------------------------------------------------------------
Server Version: WI-V2.5.9.27139 Firebird 2.5
Server Implementation: Firebird/x86-64/Windows NT
Service Version: 2**

DDL below:
**CREATE TABLE LOG_INFO EXTERNAL 'C:\ExampleDb\SomeFile.TXT'
(
  DHR timestamp DEFAULT current_timestamp,
  Log_Entry char(300) COLLATE UNICODE_CI
);**

**GRANT INSERT  ON LOG_INFO TO PROCEDURE INSERT_LOG;**

**GRANT DELETE, INSERT, REFERENCES, SELECT, UPDATE
 ON LOG_INFO TO  SYSDBA WITH GRANT OPTION;**

**Comment by Jdochoa** [comment]
Fixed, please wait the next snapshot.

**Reply by FabricioAraujoRJ** [reply]
Wow, that's what I call a premium fast service.
Thank you sooo much.
